api/README.md

```diff
@@ -20,3 +20,8 @@ Execute tests on the command line or through IntelliJ
 To run a specific test alone, add `--test` + the \
 [fully qualified class name](https://docs.gradle.org/current/userguide/java_testing.html#full_qualified_name_pattern)\
 to the command. For example, the command for running AppTest.kt would be: `./api/gradlew -p api/app :app:test --tests AppTest`
+
+## Building a docker image
+1. `./api/scripts/build` builds a docker image
+2. `./api/scripts/build-and-push` builds and pushes an image to dockerhub and runs on CI
```

Comment:
Not sure what "..and runs on CI" means here - the last thing the script does does is push the image

Reply:
Ha, I meant that this script is run on CI! Will rephrase :)